src/main.rs

```diff
@@ -4,15 +4,37 @@ use anyhow::{self, Context};
 use clap::Parser;
 use tokio::{select, signal};
 use tracing_subscriber::prelude::*;
-use wasmcloud_host::WasmbusHostConfig;
+
+mod config;
```

Comment:
nit; could you move this above the `use` block, that's so far the consistent style used throughout the codebase in this repository.

(It should be made a guideline)

Reply:
Sure